cabal-install/src/Distribution/Client/Main.hs

```diff
@@ -433,10 +435,11 @@ mainWorker args = do
       , regularCmd genBoundsCommand genBoundsAction
       , regularCmd CmdOutdated.outdatedCommand CmdOutdated.outdatedAction
       , wrapperCmd hscolourCommand hscolourCommonFlags
-      , hiddenCmd formatCommand formatAction
+      , regularCmd formatCommand formatAction
       , hiddenCmd actAsSetupCommand actAsSetupAction
       , hiddenCmd manpageCommand (manpageAction commandSpecs)
       , regularCmd CmdListBin.listbinCommand CmdListBin.listbinAction
+      , hiddenCmd dumpPackageDescriptionCommand dumpPackageDescriptionAction
       ]
         ++ concat
           [ newCmd CmdConfigure.configureCommand CmdConfigure.configureAction
```

Comment:
by moving format from hidden to regular doesn't this make things worse, not better?